biogrid/libs/biogrid-simulator/src/lib/biobrain/biobrain-class.ts

```diff
@@ -1,3 +1,18 @@
+/**
+ * @file defines the @class BioBrain which describes the brain of the grid responsible for deciding what action the grid should take.
+ * It @implements @interface Brain which add the functionalities the brain
+ * This class implements a singleton of the Brain since the grid only has one brain. it also implements:
+ * 1. @method computeAction which computes the decisions on how the power should be supplied. It takes in the state graph and 
+ * determines which grid item requires power and which one can supply it. It does the computations while minimizing the distance to
+ * be travelled to transport this power so that it minimizes power leakages
+ * 2. It also calculates @param efficiency, which is inversely proportional to power loss in the grid
```

Comment:
I think its okay to remove the specific method comments, as these might change over time. Let's leave lines 1-4, and remove `it also implements: ...` and remove lines 5-8 maybe?

Reply:
yeah, that makes sense since the class methods might change